python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -504,6 +508,24 @@ def __instancecheck__(self, __instance: Any) -> bool:
             return issubclass(type(__instance), self)
         return False
 
+    @classmethod
+    def set_transfer_blocking(cls, transfer_block: _State | None = None):
+        """Set transfer blocking flag for all instances of this proxy type.
+
+        Parameters
+        ----------
+        transfer_block : _State | None
+            None: No blocking (default)
+            _State.FAST: Block fast-to-slow transfers only
+            _State.SLOW: Block slow-to-fast transfers only
```

Comment:
Can we use a new enum with more descriptive names? We are definitely going to forget whether `_State.<STATE>` means that transferring _from_ `STATE` or _to_ `STATE` is blocked.